src/ecephys_etl/modules/align_timestamps/channel_states.py

```diff
@@ -44,12 +43,11 @@ def extract_splits_from_states(
         Sample index of each event.
     sampling_rate : numeric
         Samples / second
-    **barcode_kwargs : 
+    **barcode_kwargs :
```

Comment:
Can we refer to a different docstring that explicitly defines the expected kwargs?